beacon_node/lighthouse_network/src/peer_manager/peerdb/peer_info.rs

```diff
@@ -477,7 +474,9 @@ pub enum ConnectionDirection {
 }
 
 /// Connection Status of the peer.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(tag = "type")]
```

Comment:
I feel like we may as well use the same name we used previously

```suggestion
#[serde(tag = "status")]
```